doc/tutorials/nires_howto.rst

```diff
@@ -0,0 +1,396 @@
+observations is longer than expected, and the dither pattern of the science
+frames are set to MANUAL.  All of this means you need to edit the pypeit file to
+correct specific errors.  The corrections needed are:
+
+    - Set the frametypes for ``s190519_0059.fits`` and ``s190519_0060.fits`` to
+      ``arc,standard,tilt`` (i.e., change ``science`` to ``standard`` for these
+      frames).
+
+    - Set the ``bkg_id`` for the ``standard`` and ``science`` frames; see below
+      and :ref:`a-b_differencing`.
+
+The corrected version looks like this (pulled directly from the :ref:`dev-suite`):
+
+.. include:: ../include/keck_nires_A_corrected.pypeit.rst
+
+Use of the standard frame
+-------------------------
+
+By setting the science and standard frames to also be ``arc`` and ``tilt``
+frames, we're indicating that the OH sky lines should be used to perform the
+wavelength calibration.  The wavelength calibration for these two sets of frames
+will be performed *independently* because we assign them to different
+:ref:`calibration-groups`.
+
+The observations of the standard star in this dataset are sufficiently long that
+there is good signal in the sky lines that allows for use of the OH lines for
+its wavelength calibration.  More generally, beware that short exposures may not
+allow for this, in which case you should not designate your ``standard`` frames
+as also being ``arc,tilt`` frames and you must assign the standard frames to a
+calibration group that includes ``arc,tilt`` frames.  In this example, that
+would mean assigning all 4 science and standard frames to the same calibration
+group.
+
+The main data-reduction script (:ref:`run-pypeit`) does *not* perform the
+telluric correction; this is done by a separate script.  However, and even if
+you don't intend to telluric-correct or flux-calibrate your data, it's useful to
+include the standard star observations along with the reductions of your main
+science target, particularly if the science target is faint.  If your object is
+faint, tracing the object spectrum for extraction can be difficult using only
+the signal from the source itself.  PypeIt will resort to a "tracing crutch" if
+the source signal becomes too weak.  Without the bright standard star trace, the
+tracing crutch used is the slit/order edge, which will not include the effects
+of differential atmospheric refraction on the object centroid and therefore
+yield a poorer spectral extraction.
+
+Dither sequence
+---------------
+
+As stated above, the automated algorithms in :ref:`pypeit_setup` don't correctly
+set the :ref:`a-b_differencing` pattern for this dataset, handled by the
+``comb_id`` and ``bkg_id`` columns.  This dataset is a simple case were we have
+observed two offset positions for both the science object and the standard star.
+The numbers in the ``comb_id`` and ``bkg_id`` columns (in the corrected
+:ref:`pypeit_file`) indicate that we create both A-B and B-A images for both the
+standard and science frames.
```

Comment:
Can we add a touch more here or provide yet another urge to look at the `a-b differencing` docs?

Reply:
Added